docs/docs/frigate/installation.md

```diff
@@ -94,6 +94,21 @@ By default, the Raspberry Pi limits the amount of memory available to the GPU. I
 
 Additionally, the USB Coral draws a considerable amount of power. If using any other USB devices such as an SSD, you will experience instability due to the Pi not providing enough power to USB devices. You will need to purchase an external USB hub with it's own power supply. Some have reported success with <a href="https://amzn.to/3a2mH0P" target="_blank" rel="nofollow noopener sponsored">this</a> (affiliate link).
 
+### Hailo-8L
+
+The Hailo-8L is an M.2 card typically connected to a carrier board for PCIe, which then connects to the Raspberry Pi 5 as part of the AI Kit. However, it can also be used on other boards equipped with an M.2 M key edge connector.
+
+#### Installation 
+
+For Raspberry Pi 5 users with the AI Kit, installation is straightforward. Simply follow this [guide](https://www.raspberrypi.com/documentation/accessories/ai-kit.html#ai-kit-installation) to install the driver and software.
+
+For other boards, follow these steps for installation:
+
+1. Install the driver from the Hailo GitHub repository. A convenient script for Linux is available to clone the repository, build the driver, and install it.
+2. Copy or download [this script](https://gist.github.com/spanner3003/4b85751d671d4ac55f926e564f1abc3e#file-install_hailo8l_driver-sh).
+3. Ensure it has execution permissions with `sudo chmod +x install_hailo_driver.sh`.
+4. Run the script with `./install_hailo_driver.sh`.
+
```

Comment:
there should be a certain device that needs to be mounted into the docker container right?

Reply:
yes, `/dev/hailo0` I've updated the installation.md for better clarity.